setup.py

```diff
@@ -18,7 +18,7 @@ def check_setuptools_features():
     import pkg_resources
     try:
         list(pkg_resources.parse_requirements('foo~=1.0'))
-    except pkg_resources.RequirementParseError:
+    except ValueError:
         exit('Your Python distribution comes with an incompatible version '
              'of `setuptools`. Please run:\n'
              ' $ pip install --update setuptools\n'
```

Comment:
It should be `--upgrade` but I'm not sure `pip3` is necessary. The BigchainDB installation documentation and the AWS deployment fabfile both use `pip` to do the upgrade and it doesn't seem to cause any problems. If we do decide to go with `pip3` then I can just change the documentation and AWS deployment fabfile.